src/common/constants.js

```diff
@@ -1,2 +1,34 @@
 export const PAGES = ["Home", "Products", "About"];
 export const SETTINGS = ["Profile", "Account", "Dashboard", "Logout"];
+export const SLIDER_IMAGES = [
+  {
+    label: "xiaomi",
+    imgPath:
+      "https://www.ctrs.com.ua/_next/image/?url=https://i.citrus.world/uploads/shares/2022/november/nerealnye-vyhodnye-realme/1066x400.jpg?_t=1668591669&w=1080&q=100",
```

Comment:
someday these pictures will not be. You need to add a default image if there is no image on the server